From 9e065854a5c1fdbc063a5c8b59f2615d9dffff0d Mon Sep 17 00:00:00 2001
From: Li Huang <putin.li@rock-chips.com>
Date: Wed, 1 Aug 2018 21:45:27 +0800
Subject: [PATCH] video/rockchip: rga: Remove MMU struct unuse member

it will set alpha_rop_mode to 0, fixup rga blend error.

Change-Id: I6c10df0e4d198ab8da8d352239151ddc3605e860
Signed-off-by: Li Huang <putin.li@rock-chips.com>
---
 drivers/video/rockchip/rga/rga.h | 10 ----------
 1 file changed, 10 deletions(-)

diff --git a/drivers/video/rockchip/rga/rga.h b/drivers/video/rockchip/rga/rga.h
index 4f6ccedf803d..8a337b86ad63 100644
--- a/drivers/video/rockchip/rga/rga.h
+++ b/drivers/video/rockchip/rga/rga.h
@@ -187,16 +187,6 @@ typedef struct MMU
     unsigned char mmu_en;
     unsigned long base_addr;
 	uint32_t mmu_flag;
-
-	unsigned long src0_base_addr;
-	unsigned long src1_base_addr;
-	unsigned long dst_base_addr;
-	unsigned long els_base_addr;
-
-	u8 src0_mmu_flag;
-	u8 src1_mmu_flag;
-	u8 dst_mmu_flag;
-	u8 els_mmu_flag;
 } MMU;
 
 
-- 
2.35.3

